modules/virt-vm-fields-web.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+// VM wizard includes additional options to VM template wizard
+// Call appropriate attribute in the assembly
+
+[id="virt-vm-fields-web_{context}"]
+= Virtual machine web console fields
+
+The following table outlines the editable fields when creating a new virtual machine (VM) in the {VirtProductName} web console.
```

Comment:
Now I'm confused. Why are there 2 modules with basically the same content? I think we should use this one, for both editing and creating a VM.